README.md

```diff
@@ -182,10 +182,10 @@ In the second Fee Provider mode, the Fee Provider module pays a fee instead of t
 1. Set like below
 
 ```ruby
-Glurby.configuration do |c|
+Glurby.configure do |config|
   # Use FeeProvider to supply inputs for fees on each transaction that is created on Glueby.
-  c.fee_provider_bears!
-  c.fee_provider_config = {
+  config.fee_provider_bears!
```

Comment:
Why destructive method?

Reply:
This is going to modify the configuration itself.

Reply:
Isn't it natural for the configuration to change since it is a method that does the configuration?

What is the difference from adapter configuration?

Reply:
The adapter configuration method `def wallet_adapter=(adapter)` uses assignments style. It is explicit about modifying the configuration. So we don't need to use the destructive method style.

I referred Rspec::Core configuration class like here https://github.com/rspec/rspec-core/blob/main/lib/rspec/core/configuration.rb#L1844 

In RSpec configuration, the methods that don't show it is destructive clearly use the destructive method style.
The methods using assignment style or the name has word show modification like 'add' don't use destructive style.